03.bowling/bowling.rb

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
+
+frames = []
+shots.each_slice(2) do |shot|
+  frames << shot
+end
+
+point = 0
+frames.each_with_index do |frame, idx|
+  first_next_frame = frames[idx + 1]
+  second_next_frame = frames[idx + 2]
+  strike = frame[0] == 10
+  spare = frame.sum == 10
```

Comment:
```suggestion
  spare = !strike && frame.sum == 10
```

念のためストライクとスペアは区別した方がいいかも

Reply:
修正しました。